lib/license_scout/overrides.rb

```diff
@@ -671,6 +671,7 @@ def default_overrides
         ["debug", "MIT", ["https://raw.githubusercontent.com/visionmedia/debug/master/LICENSE"]],
         ["performance-now", "MIT", ["https://raw.githubusercontent.com/braveg1rl/performance-now/master/license.txt"]],
         ["cli-table", "MIT", ["README.md"]],
+        ["process", "MIT", ["https://raw.githubusercontent.com/defunctzombie/node-process/master/LICENSE"]],
```

Comment:
Guessing this is a zombie comment, but generally we use master since license scout can't match the license file URL to the version you actually have.